src/components/Tooltip.js

```diff
@@ -79,7 +79,13 @@ class Tooltip extends PureComponent {
      * @returns {Promise}
      */
     getWrapperPosition() {
-        return new Promise((resolve => this.wrapperView.measureInWindow((x, y) => resolve({x, y}))));
+        return new Promise(((resolve, reject) => {
```

Comment:
This change was just to suppress a console warning: `cannot call method measureInWindow of undefined`